node-notes-cli/app.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+
+const jsonFile = require('./data.json');
+const currentID = jsonFile.nextId;
+
+function read() {
+  for (const entry in jsonFile.notes) {
+    console.log(JSON.stringify(`${entry}: ${jsonFile.notes[entry]}`));
+  }
+}
+
+function create() {
+  jsonFile.notes[currentID] = process.argv[3];
+  console.log(jsonFile.notes);
```

Comment:
Please remove the logging from your write operations:

---

![image](https://user-images.githubusercontent.com/16143771/214219283-1edf45dd-5520-4b43-a151-a5b9ba312f43.png)